docs/getting-started/bot.md

```diff
@@ -0,0 +1,412 @@
+Atomist Bot does not need to ask you for the repository name, it
+already knows and supplies that information to the command handler
+automatically.
+
+As usual, your Atomist Bot will start a thread to collect all the
+information necessary it needs to complete the request, creating a new
+issue on the `sprocket` project.
+
+<div class="ss-container">
+  <img src="../images/create-issue-flow.png" alt="Creating an Issue" class="ss-small">
+</div>
+
+Once you have entered all the needed information, your Atomist Bot
+will respond with a summary and the option to submit or cancel.
+
+<div class="ss-container">
+  <img src="../images/create-issue-dialog.png" alt="Issue Summary" class="ss-small">
+</div>
+
+Click the "Submit" button and Atomist will create the issue and then
+respond back in the main `#sprocket` channel.
+
+<div class="ss-container">
+  <img src="../images/successfully-created-issue-response.png" alt="Issue Created" class="ss-medium">
+</div>
+
+But that's not where the interaction stops.  The Atomist Bot will also
+post a message in the `#sprocket` channel containing a summary of the
+newly created issue *and* some buttons for common actions taken on
+issues: assign it someone, add a label, add a comment, etc.
+
+<div class="ss-container">
+  <img src="../images/new-issue-follow-on-buttons.png" alt="Issue Created" class="ss-medium">
+</div>
+
+Go ahead and click on the link to see the issue on GitHub.
+
+<div class="ss-container">
+  <img src="../images/new-issue-in-github.png" alt="Issue in GitHub" class="ss-large">
+</div>
+
+Now head back to the `#sprocket` channel in Slack and click on the
+"Bug" button to add the "bug" label to the issue:
+
+<div class="ss-container">
+  <img src="../images/issue-labelled-as-bug-response.png" alt="Issue labelled as a Bug" class="ss-medium">
+</div>
+
+We see the Atomist Bot responding that it has successfully edited the
+issue.  Let's make sure.  Click on the link to open up the issue in
+GitHub.
+
+<div class="ss-container">
+  <img src="../images/issue-in-github.png" alt="Issue in GitHub" class="ss-large">
+</div>
+
+We can see that Atomist, as us, has labeled the issue a bug, just as
+we asked.
+
+Now imagine that you are another team member who has noticed this new
+issue and wants to add a comment from inside GitHub.  Using the GitHub
+web interface, add a comment and click the "Comment" button.
+
+<div class="ss-container">
+  <img src="../images/new-issue-comment-in-github.png" alt="New comment on issue in GitHub" class="ss-large">
+</div>
+
+You will see a new message in the `#sprocket` channel from the Atomist
+Bot notifying you that the issue has been updated.
+
+<div class="ss-container">
+  <img src="../images/new-comment-notification-in-slack.png" alt="New comment notification in Slack" class="ss-medium">
+</div>
+
+The circle is complete!  After connecting Atomist and GitHub, we can
+see GitHub events in Slack, take action on them, and see the result in
+GitHub and Slack.
+
```

Comment:
Note that is really just an reworking to clarify what was already here. Many more improvements can be made, especially given the introduction of tutorials since this text was reworked for this PR.